lib/index.js

```diff
@@ -100,9 +92,8 @@ api.canonize = util.callbackify(async function(dataset, options) {
  * Synchronously canonizes an RDF dataset.
  *
  * @param dataset the dataset to canonize.
- * @param [options] the options to use:
- *          [algorithm] the canonicalization algorithm to use, `URDNA2015` or
- *            `URGNA2012` (default: `URGNA2012`).
+ * @param options the options to use:
+ *          algorithm the canonicalization algorithm to use: `URDNA2015`.
```

Comment:
`algorithm` is required now, and there is no default?

Reply:
I think the docs were just wrong before.

Reply:
Yeah, I was just making the code match the docs.  Same in rdf-canonize.